_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,405 @@
+            <input class="form-input input-text input-alt has-error" type="text" name="full_name_simple_err" id="full_name_simple_err" placeholder="e.g. Jane Doe" required>
+        </div>
+        <div class="field">
+            <label class="form-label" for="full_name_simple_suc">Your name (simple text field, success)</label>
+            <input class="form-input input-text input-alt has-success" type="text" name="full_name_simple_suc" id="full_name_simple_suc" placeholder="e.g. Jane Doe" required>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6 example-set-hd">Select/Dropdown/Combo</h4>
+<div class="example-set">
+    <fieldset class="field-group">
+        <legend class="field-group-hd sr-only">Some select menus</legend>
+        <div class="field">
+            <label class="form-label" for="select_01">Default select menu</label>
+            <select class="form-input input-select" name="select_01" id="select_01">
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+        <div class="field">
+            <label class="form-label" for="select_01_err">Default select menu with error</label>
+            <select class="form-input input-select has-error" name="select_01_err" id="select_01_err">
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+        <div class="field">
+            <label class="form-label" for="select_01_suc">Default select menu with success</label>
+            <select class="form-input input-select has-success" name="select_01_suc" id="select_01_suc">
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+
+        <div class="field">
+            <label class="form-label" for="select_02">Enhanced select menu</label>
```

Comment:
Please sync up your label naming convention - "Real label for example field (example explanation)".